### PR TITLE
[DLS-6490] rename host key to 'if'

### PR DESCRIPTION
**Pull request by ruthdas03**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -134,7 +134,7 @@ microservice {
       originatorId = "DA_PTA"
     }
 
-    paye-personal-details-if {
+    if {
```
Comment (johno1985) [inline conf/application.conf:137]: merge with if block above

**Review by johno1985 — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (ruthdas03) [reply conf/application.conf]: Done

**Comment by johno1985** [comment]
please squash

**Comment by ruthdas03** [comment]
> please squash



> please squash

Done

**Review by johno1985 — APPROVED** [review]

**Pull request merged**